<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScatterPlotDialog</class>
 <widget class="QDialog" name="ScatterPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scatter Plot</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_x">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_y">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QToolButton" name="filter_chart">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="app.qrc">
         <normaloff>:/icons/view-filter.png</normaloff>
         <normalon>:/icons/view-no-filter.png</normalon>:/icons/view-filter.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ChartView" name="chart"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChartView</class>
   <extends>QGraphicsView</extends>
   <header>chartview.h</header>
   <slots>
    <signal>selected(float,float,bool)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filter_chart</sender>
   <signal>clicked()</signal>
   <receiver>ScatterPlotDialog</receiver>
   <slot>updateChart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>284</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateChart()</slot>
 </slots>
</ui>
